### DISCUSSION TITLE
Freetext search

### DISCUSSION THREAD

**Discussion by Caeylinity** [NilsSve/DfRefactor#23] [Ideas]
I would like to be able to search for a specific function in the Functions List tab. A freetext search option based on Function Name and maybe Description would be very helpful, especially as more and more refactoring functions get added.

**Comment by NilsSve** [comment]
I can see how that would be very useful, and not too hard to implement.

**Comment by NilsSve** [comment]
A solution has been implemented, via the standard DataFlex selection list.

The problem previously when trying to find a function in the popup selection list was that the Functions data table was missing an "Ignore Casing" for the Function_Name column.

Now the selection list works as it should, and that will have to do for now.
